Gopkg.lock

```diff
@@ -163,6 +163,12 @@
   revision = "202f25545ea4cf9b191ff7f846df5d87c9382c2b"
   version = "v1.0.0"
 
+[[projects]]
+  name = "github.com/spf13/pflag"
+  packages = ["."]
+  revision = "298182f68c66c05229eb03ac171abe6e309ee79a"
+  version = "v1.0.3"
+
```

Comment:
@chmouel This gets generated with `dep ensure`. It not being used in idler, but it is one of the dependencies in `k8s.io/apimachinery`. So, I think that's why it is here.